lib/rules/jsx-indent-props.js

```diff
@@ -174,7 +174,8 @@ module.exports = {
     function checkNodesIndent(nodes, indent) {
       nodes.forEach((node) => {
         const nodeIndent = getNodeIndent(node);
-        if (line.isUsingOperator && !line.currentOperator && indentSize !== 'first' && !ignoreTernaryOperator) {
+        if (
+          line.isUsingOperator && !line.currentOperator && indentSize !== 'first' && !ignoreTernaryOperator) {
```

Comment:
i don't think this is more readable as-is. 

Generally, if the entire if condition can't fit on one line, then each condition needs to be on its own line:
```suggestion
        if (
          line.isUsingOperator
          && !line.currentOperator
          && indentSize !== 'first'
          && !ignoreTernaryOperator
        ) {
```